### DISCUSSION TITLE
/t deposit not working

### DISCUSSION THREAD

**Discussion by ardacy2112** [TownyAdvanced/Towny#7585] [Q&A] [answered]
No money deposited in the bank

**Comment by LlmDl** [comment, marked as answer]
Your economy plugin probably does not fully support the vault API.

See the FAQ for a list of eco plugins to not use.

**Reply by ardacy2112** [reply]
Do you need a plugin for Deposit?
I didn't know, but I'd appreciate a link to which plugin you need <3

**Reply by Warriorrrr** [reply]
You can check our [faq](https://github.com/TownyAdvanced/Towny/wiki/Frequently-Asked-Questions)

**Comment by ardacy2112** [comment]
It doesn't work again, and this Towny plugin has made all the commands like itself, now when I type time set day, it is not the game but the commands of the towny, in short, it has taken over the commands.

**Reply by LlmDl** [reply]
I don't really understand your translation. Towny will attempt to take over the /t command but won't override /time.

**Comment by ardacy2112** [comment]
İ install the new wconomy and solved.
